### PR TITLE
Update test.ts

### PR DESCRIPTION
**Pull request by hunterliu1003**

### CODE REVIEW

**Review by Mini-ghost — CHANGES_REQUESTED** [review]
test

**Review by Mini-ghost — COMMENTED** [review]
這是一搬回覆

**Review by Mini-ghost — APPROVED** [review]

**Review by Mini-ghost — CHANGES_REQUESTED** [review]
Request changes

File: test.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// test
```
Comment (hunterliu1003) [inline test.ts:1]: review 1

File: test.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// test
+// test2
```
Comment (hunterliu1003) [inline test.ts:2]: review 2

**Pull request merged**